school_lunch/i18n/fr_BE.po

```diff
@@ -751,6 +876,16 @@ msgstr "Envoyer l'email des repas"
 msgid "Send Now"
 msgstr "Envoyer maintenant"
 
+#. module: school_lunch
+#: model_terms:ir.ui.view,arch_db:school_lunch.gift_card_comment_view_search
+msgid "Sent"
+msgstr "Envoyé"
+
+#. module: school_lunch
+#: model_terms:ir.ui.view,arch_db:school_lunch.website_month_display
+msgid "Septembre"
+msgstr ""
```

Comment:
The source text looks already FR, I would remove this entry since it's also untranslated.

There are many more above, and IMO they should all be removed since the source text is not English and have no translation.

It's okay to update the PO file from the client db, but before committing it it should be reviewed and make sense for what a PO file should do.

Reply:
I think they are present when you extract the file from the db.

Reply:
> I think they are present when you extract the file from the db.

Yes, but that doesn't necessarily mean we should include them if they're not doing what they should. There's a lot of "garbage" data in dbs in general.